src/main/java/de/doubleslash/keeptime/controller/Controller.java

```diff
@@ -167,18 +165,20 @@ private boolean isProjectActive(final Project p) {
       return p == model.activeWorkItem.get().getProject();
    }
 
-   public void editProject(final Project p, final String newName, final Color newColor, final boolean isWork,
-         final int newIndex) {
-      LOG.info("Changing project '{}' to '{}' '{}' '{}'", p, newName, newColor, isWork);
+   public void editProject(final Project projectToBeUpdated, final Project newValuedProject) {
+      LOG.info("Changing project '{}' to '{}' '{}' '{}'", projectToBeUpdated, newValuedProject.getName(),
```

Comment:
maybe just print the two projects, they implement toString, which should be sufficient

Reply:
Done.